src/components/select/_select.scss

```diff
@@ -121,14 +121,21 @@
   }
 
   .#{$prefix}--select--inline {
-    display: flex;
-    flex-direction: row;
-    align-items: center;
+    display: grid;
+    grid-template-columns: auto auto;
```

Comment:
Are there any design guidelines that state what the maximum length of the label and/or input field should be (or some kind of ratio between label length and input field length)? If so the `minmax` function may be useful here

Reply:
I don't believe so, the recommendation for inline select is to be used when there are multiple in a row. I don't believe having the label to the left is a standard use-case that is recommended. http://www.carbondesignsystem.com/components/select/usage